credential-provider/src/common/helpers.ts

```diff
@@ -14,3 +14,11 @@ export const addYears = (date: Date, years: number): Date => {
   date.setFullYear(date.getFullYear() + years);
   return date;
 };
+
+export const createResponseToken = (token: string): string => {
```

Comment:
Would be good to have inline documentation here. What is this fn doing and what sort of token is it? Looks like it "parses" a token rather than "creating" it.

Reply:
I haven't done making changes yet, it's just a placeholder. Will send you a flow diagram on slack in a min